1-mandatory/2-people-i-know.js

```diff
@@ -377,14 +377,14 @@ In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
 
-let thirtyFiveOrOlder = [];
+let thirtyFiveOrOlder = friends.filter(friend => friend.age > 35);
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
-let powerNetEmails = [];
+let powerNetEmails = friends.filter(friend => friend.company === "POWERNET").map(friend => friend.email);
```

Comment:
Comment: Well done

Consider: Another way - 
let powerNetEmails = friends
  .filter((friend) => friend.company.includes(`POWERNET`))
  .map((friend) => friend.email);

Reply:
Thank you